manage

```diff
@@ -499,12 +499,14 @@ if ! isInstalled ${SED_EXE}; then
 
   cat <<-EOF
   The recommended approach to installing the required package(s) is to use either [Homebrew](https://brew.sh/) (MAC)
-  or [Chocolatey](https://chocolatey.org/) (Windows).  For more information visit https://mikefarah.github.io/yq/
+  or [Chocolatey](https://chocolatey.org/) (Windows) or [GitHub](https://github.com/mirror/sed) (Linux).  For more information visit https://mikefarah.github.io/yq/
```

Comment:
```suggestion
  or [Chocolatey](https://chocolatey.org/) (Windows) or [GitHub](https://github.com/mirror/sed) (Linux).  For more information visit https://www.gnu.org/software/sed/
```

Reply:
You still need to change the link reference from `https://mikefarah.github.io/yq/` to `https://mikefarah.github.io/yq/`